### PR TITLE
docs: add Chinese version of control-api.md

### LINKED ISSUE #5006: docs: do we need a Chinese version of control-api.md?
if yes,I'd like to give a try

### PR DESCRIPTION
**Pull request by RedemptionC**
### What this PR does / why we need it:

this PR adds a Chinese version of control-API.md
resolve #5006 

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Comment by RedemptionC** [comment]
@tzssangglass @tokers 
Hello,I have submitted a version of translation,you could have a check
Also,I don't know that this sentence mean
`If a plugin adds such a control API, please refer to each plugin's documentation for those APIs.`
could you provide some explanations?

**Comment by RedemptionC** [comment]
@tzssangglass @tokers
thanks to @spacewander , I have completed this vesion of translation
you could check again

File: docs/zh/latest/control-api.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+control API 可以被用来:
+
+* 暴露APISIX内部状态信息
+* 控制单个APISIX的数据平面的行为
```
Comment (tzssangglass) [inline docs/zh/latest/control-api.md:27]:
```suggestion
* 暴露 APISIX 内部状态信息
* 控制单个 APISIX 的数据平面的行为
```

File: docs/zh/latest/control-api.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+默认情况下,control API是启用的,监听`127.0.0.1:9090`.你可以通过修改`apisix/conf/config.yaml`中的control部分来更改设置,如下:
```
Comment (tzssangglass) [inline docs/zh/latest/control-api.md:29]:
```suggestion
默认情况下，control API 是启用的，监听 `127.0.0.1:9090`。你可以通过修改 `apisix/conf/config.yaml` 中的control 部分来更改设置，如下:
```

And please fix other similar errors.

- Chinese should be preceded by spaces between English, numbers and other special symbols.
- Use Chinese punctuation.

**Comment by RedemptionC** [comment]
@tzssangglass hello,I have commited changes . you can check again

**Review by iamayushdas — APPROVED** [review]

File: docs/zh/latest/control-api.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+# limitations under the License.
+#
+-->
+
+control API 可以被用来：
+
+* 暴露 APISIX 内部状态信息
+* 控制单个 APISIX 的数据平面的行为
+
+默认情况下，control API 是启用的，监听`127.0.0.1:9090`。你可以通过修改`apisix/conf/config.yaml`中的 control 部分来更改设置，如下：
```
Comment (tzssangglass) [inline docs/zh/latest/control-api.md:29]:
```suggestion
默认情况下，control API 是启用的，监听 `127.0.0.1:9090`。你可以通过修改 `apisix/conf/config.yaml` 中的 control 部分来更改设置，如下：
```

File: docs/zh/latest/control-api.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+```yaml
+apisix:
+  ...
+  enable_control: true
+  control:
+    ip: "127.0.0.1"
+    port: 9090
+```
+
+注意: control API server 不应该被配置成监听公共流量(public traffic)
```
Comment (tzssangglass) [inline docs/zh/latest/control-api.md:40]:
```suggestion
注意: control API server 不应该被配置成监听公共流量(public traffic)。
```

File: docs/zh/latest/control-api.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+
+插件被启用时可以添加自己的 control API。如果你对他们感兴趣，请参阅对应插件的文档。
+
+## 独立于插件的 control API
+
+以下是支持的API:
+
+### GET /v1/schema
+
+引入自2.2版本
```
Comment (tzssangglass) [inline docs/zh/latest/control-api.md:52]:
```suggestion
引入自 2.2 版本
```

File: docs/zh/latest/control-api.md [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+    "stream-plugins": {
+        "mqtt-proxy": {
+            ...
+        },
+        ...
+    }
+}
+```
+
+返回结果中`plugins`部分，只有启用了的插件才会被包含在内。(返回结果中的)一些插件可能会缺失如`consumer_schema`或者`type`字段,这取决于插件的定义
```
Comment (tzssangglass) [inline docs/zh/latest/control-api.md:87]:
```suggestion
只有启用了的插件才会被包含在返回结果中 `plugins` 部分。(返回结果中的)一些插件可能会缺失如 `consumer_schema` 或者 `type` 字段，这取决于插件的定义
```

File: docs/zh/latest/control-api.md [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+
+### GET /v1/healthcheck
+
+引入自2.3版本
```
Comment (tzssangglass) [inline docs/zh/latest/control-api.md:91]:
```suggestion
引入自 2.3 版本
```

File: docs/zh/latest/control-api.md [hunk 8]
```diff
@@ -0,0 +1,202 @@
...
+
+使用以下格式返回当前的[health check](health-check.md)状态
```
Comment (tzssangglass) [inline docs/zh/latest/control-api.md:93]:
```suggestion
使用以下格式返回当前的 [health check](health-check.md) 状态
```

File: docs/zh/latest/control-api.md [hunk 9]
```diff
@@ -0,0 +1,202 @@
...
+        }
+    ],
+    "src_id": "1",
+    "src_type": "upstreams"
+}
+```
+
+### POST /v1/gc
+
+引入自2.8版本
```
Comment (tzssangglass) [inline docs/zh/latest/control-api.md:198]:
```suggestion
引入自 2.8 版本
```

File: docs/zh/latest/control-api.md [hunk 10]
```diff
@@ -0,0 +1,202 @@
...
+
+在 http 子系统中触发一次完整的GC
```
Comment (tzssangglass) [inline docs/zh/latest/control-api.md:200]:
```suggestion
在 http 子系统中触发一次完整的 GC
```

File: docs/zh/latest/control-api.md [hunk 11]
```diff
@@ -0,0 +1,202 @@
...
+
+注意，当你启用 stream proxy 时，APISIX 将为 stream 子系统运行另一个 Lua 虚拟机。不会触发这个Lua虚拟机中的完整 GC
```
Comment (tzssangglass) [inline docs/zh/latest/control-api.md:202]:
```suggestion
注意，当你启用 stream proxy 时，APISIX 将为 stream 子系统运行另一个 Lua 虚拟机。不会触发这个 Lua 虚拟机中的全量 GC。
```

File: docs/zh/latest/control-api.md [hunk 12]
```diff
@@ -0,0 +1,202 @@
...
+
+* src_type：表示 health checker 的来源。值是`routes,services,upstreams`其中之一
+* src_id：表示创建health checker的对象的id。例如，假设id为1的Upstream对象创建了一个health checker，那么`src_type`就是`upstreams`，`src_id`就是1
+* name： 表示 health checker 的名称
+* nodes： health checker 的目标节点
+* healthy_nodes： 表示 health checker 检测到的健康节点
+
+用户也可以通过`/v1/healthcheck/$src_type/$src_id`来获取指定 health checker 的状态
+
+例如,`GET /v1/healthcheck/upstreams/1`返回:
```
Comment (tzssangglass) [inline docs/zh/latest/control-api.md:164]:
```suggestion
例如，`GET /v1/healthcheck/upstreams/1` 返回：
```

File: docs/zh/latest/control-api.md [hunk 13]
```diff
@@ -0,0 +1,202 @@
...
+
+每个entry包含以下字段：
+
+* src_type：表示 health checker 的来源。值是`routes,services,upstreams`其中之一
+* src_id：表示创建health checker的对象的id。例如，假设id为1的Upstream对象创建了一个health checker，那么`src_type`就是`upstreams`，`src_id`就是1
+* name： 表示 health checker 的名称
+* nodes： health checker 的目标节点
+* healthy_nodes： 表示 health checker 检测到的健康节点
+
+用户也可以通过`/v1/healthcheck/$src_type/$src_id`来获取指定 health checker 的状态
```
Comment (tzssangglass) [inline docs/zh/latest/control-api.md:162]:
```suggestion
用户也可以通过 `/v1/healthcheck/$src_type/$src_id` 来获取指定 health checker 的状态。
```

File: docs/zh/latest/control-api.md [hunk 14]
```diff
@@ -0,0 +1,202 @@
...
+        ],
+        "src_id": "1",
+        "src_type": "routes"
+    }
+]
+```
+
+每个entry包含以下字段：
+
+* src_type：表示 health checker 的来源。值是`routes,services,upstreams`其中之一
```
Comment (tzssangglass) [inline docs/zh/latest/control-api.md:156]:
```suggestion
* src_type：表示 health checker 的来源。值是 `routes,services,upstreams`其中之一
```

File: docs/zh/latest/control-api.md [hunk 15]
```diff
@@ -0,0 +1,202 @@
...
+* src_id：表示创建health checker的对象的id。例如，假设id为1的Upstream对象创建了一个health checker，那么`src_type`就是`upstreams`，`src_id`就是1
```
Comment (tzssangglass) [inline docs/zh/latest/control-api.md:157]:
```suggestion
* src_id：表示创建 health checker 的对象的id。例如，假设 id 为 1 的 Upstream对 象创建了一个 health checker，那么 `src_type` 就是 `upstreams`，`src_id` 就是 1
```

**Review by tzssangglass — COMMENTED** [review]
and pls check for other similar errors

File: docs/zh/latest/control-api.md [hunk 16]
```diff
@@ -0,0 +1,202 @@
...
+  control:
+    ip: "127.0.0.1"
+    port: 9090
+```
+
+注意: control API server 不应该被配置成监听公共流量(public traffic)。
+
+## 通过插件添加的 control API
+
+插件被启用时可以添加自己的 control API。如果你对他们感兴趣，请参阅对应插件的文档。
```
Comment (starsz) [inline docs/zh/latest/control-api.md:44]: Maybe we can refer to this doc about the `插件文档`.

https://github.com/apache/apisix/blob/master/docs/zh/latest/plugin-develop.md#%E6%B3%A8%E5%86%8C%E6%8E%A7%E5%88%B6%E6%8E%A5%E5%8F%A3

Comment (RedemptionC) [reply docs/zh/latest/control-api.md:44]: @starsz 
do you mean that we need to change the url in plugin-develop.md for control-api.md to this Chinese version?

File: docs/zh/latest/control-api.md [hunk 17]
```diff
@@ -0,0 +1,202 @@
...
+    "stream-plugins": {
+        "mqtt-proxy": {
+            ...
+        },
+        ...
+    }
+}
+```
+
+只有启用了的插件才会被包含在返回结果中 `plugins` 部分。(返回结果中的)一些插件可能会缺失如 `consumer_schema` 或者 `type` 字段，这取决于插件的定义
```
Comment (tzssangglass) [inline docs/zh/latest/control-api.md:87]:
```suggestion
只有启用了的插件才会被包含在返回结果中 `plugins` 部分。(返回结果中的)一些插件可能会缺失如 `consumer_schema` 或者 `type` 字段，这取决于插件的定义。
```

File: docs/zh/latest/control-api.md (hunk 16 above)
Comment (starsz) [reply docs/zh/latest/control-api.md:44]: Yes. We need.

Another thing, we can add the `plugin-develop.md` link 
```
插件被启用时可以添加自己的 control API，参考[插件开发文档](plugin-develop.md)
```

**Comment by RedemptionC** [comment]
@starsz 
hello,I have made other Chinese doc refer to this version rather the English version
you could have a check

**Review by tzssangglass — APPROVED** [review]

**Review by membphis — APPROVED** [review]

File: docs/zh/latest/plugin-develop.md [hunk 18]
```diff
@@ -378,7 +378,7 @@ end
 
 ### 注册控制接口
 
-如果你只想暴露 API 到 localhost 或内网，你可以通过 [Control API](../../en/latest/control-api.md) 来暴露它。
+如果你只想暴露 API 到 localhost 或内网，你可以通过 [Control API](../../zh/latest/control-api.md) 来暴露它。
```
Comment (spacewander) [inline docs/zh/latest/plugin-develop.md:381]: We don't need to go back to the grandparent directory and then go to his children. Just do what the English version does.

File: docs/zh/latest/control-api.md [hunk 19]
```diff
@@ -0,0 +1,202 @@
...
+```yaml
+apisix:
+  ...
+  enable_control: true
+  control:
+    ip: "127.0.0.1"
+    port: 9090
+```
+
+注意: control API server 不应该被配置成监听公共流量(public traffic)。
```
Comment (spacewander) [inline docs/zh/latest/control-api.md:40]:
```suggestion
注意: control API server 不应该被配置成监听公网地址。
```

File: docs/zh/latest/control-api.md [hunk 20]
```diff
@@ -0,0 +1,202 @@
...
+
+## 通过插件添加的 control API
+
+插件被启用时可以添加自己的 control API。如果你对他们感兴趣，请参阅[插件开发文档](plugin-develop.md)。
```
Comment (spacewander) [inline docs/zh/latest/control-api.md:44]: The translation is incorrect. It is quite different from the English version

File: docs/zh/latest/control-api.md [hunk 21]
```diff
@@ -0,0 +1,202 @@
...
+        ],
+        "src_id": "1",
+        "src_type": "routes"
+    }
+]
+```
+
+每个 entry 包含以下字段：
+
+* src_type：表示 health checker 的来源。值是 `routes,services,upstreams` 其中之一
```
Comment (spacewander) [inline docs/zh/latest/control-api.md:156]: The translation is different from the English version?

File: docs/zh/latest/control-api.md [hunk 22]
```diff
@@ -0,0 +1,202 @@
...
+}
+```
+
+### POST /v1/gc
+
+引入自 2.8 版本
+
+在 http 子系统中触发一次全量 GC
+
+注意，当你启用 stream proxy 时，APISIX 将为 stream 子系统运行另一个 Lua 虚拟机。不会触发这个 Lua 虚拟机中的全量 GC。
```
Comment (spacewander) [inline docs/zh/latest/control-api.md:202]:
```suggestion
注意，当你启用 stream proxy 时，APISIX 将为 stream 子系统运行另一个 Lua 虚拟机。它不会触发这个 Lua 虚拟机中的全量 GC。
```

File: docs/zh/latest/control-api.md (hunk 21 above)
Comment (RedemptionC) [reply docs/zh/latest/control-api.md:156]: >where the health checker comes from. The value is one of `["routes", "services", "upstreams"]`.
@spacewander 
what's different?

**Review by tokers — APPROVED** [review]

**Review by starsz — APPROVED** [review]

File: docs/zh/latest/control-api.md (hunk 21 above)
Comment (spacewander) [reply docs/zh/latest/control-api.md:156]: The code is `["routes", "services", "upstreams"]` in En version, but it is `routes,services,upstreams` in Ch version.

Comment (RedemptionC) [reply docs/zh/latest/control-api.md:156]: @spacewander 
that's right,but I don't see what `[]` is needed
Of course I can add it to keep consistent with English version

**Review by spacewander — APPROVED** [review]

**Pull request merged**